code/_onclick/hud/rendering/plane_master_group.dm

```diff
@@ -93,20 +93,20 @@
 // So they look nicer. if you can't it's all good, if you think you can sanely look at monster's work
 // It's hard, and potentially expensive. be careful
 /datum/plane_master_group/proc/transform_lower_turfs(datum/hud/source, new_offset, use_scale = TRUE)
+	// Check if this feature is disabled for the client, in which case don't use scale.
+	if(!our_hud?.mymob?.client?.prefs?.read_preference(/datum/preference/toggle/multiz_parallax))
+		use_scale = FALSE
+
 	// No offset? piss off
 	if(!SSmapping.max_plane_offset)
 		return
+
 	active_offset = new_offset
+
 	// Each time we go "down" a visual z level, we'll reduce the scale by this amount
 	// Chosen because mothblocks liked it, didn't cause motion sickness while also giving a sense of height
 	var/scale_by = 0.965
-	// If our mob can see through walls
 	if(!use_scale)
-		// This is a workaround for two things
-		// First of all, if a mob can see objects but not turfs, they will not be shown the holder objects we use for
-		// What I'd like to do is revert to images if this case throws, but image vis_contents is broken
-		// https://www.byond.com/forum/post/2821969
-		// If that's ever fixed, please just use that. thanks :)
```

Comment:
please do not remove my self reminding comment

Reply:
Done